doc/source/examples/resolved-cfd-dem/sedimentation-1-particle/sedimentation-1-particle.rst

```diff
@@ -0,0 +1,345 @@
+
+Non-Linear Solver Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+	subsection non-linear solver
+	  set verbosity               = verbose
+	  set tolerance               = 1e-6
+	  set max iterations          = 10
+	  set residual precision      = 5
+	  set force rhs calculation   = true
+	end
+	
+* The ``tolerance`` is set to 1e-6. This is small enough to ensure that the flow field is adequately resolved, as here, we expect a velocity of the particle of the order of 10.
+
+* The ``max iterations`` is set to 10. The objective here is to allow enough Newton non-linear steps to ensure the convergence and limit the time pass on a time step if the system is too stiff to reach the tolerance fix.  
+
+* The ``force rhs calculation`` is set to ``true``. This is the most important modification with most of the other examples. By default, the non-linear solver will recalculate the RHS only after the update of the solution. But here, we need to evaluate it before every matrix resolution, and we cannot use the last RHS evaluation that was done after the last newton iteration. The particle position was updated between these two steps, changing the RHS evaluation. This means that for every non-linear step, we evaluate the RHS twice. The non-linear solver follows this sequence of steps for each newton iteration.
+	* update the particle position
+	* update the Jacobian matrix
+	* update the RHS
+	* solve the matrix system
+	* reevaluate the RHS to check the convergence.
+	
+	
+Linear Solver Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. code-block:: text
+
+	subsection linear solver
+	  set method                                 = gmres
+	  set max iters                              = 1000
+	  set max krylov vectors 		     = 1000
+	  set relative residual                      = 1e-4
+	  set minimum residual                       = 1e-11
+	  set ilu preconditioner fill                = 0
+	  set ilu preconditioner absolute tolerance  = 1e-20
+	  set ilu preconditioner relative tolerance  = 1.00
+	  set verbosity                              = verbose
+	end
+	
+* The ``method`` is set to ``gmres``. This solver is cheaper than the other option, and this case doesn't require any special preconditioner. This makes the ``gmres`` solver the best option available.
+
+* The ``max iters`` is set to 1000. This is a lot more steps than how much it should take to solve the system.
+
+* The ``max krylov vectors`` is set to 1000. This is to ensure that we keep the full Arnoldi basis for each new iteration. From experience keeping a maximum of Krylov vector results in a faster resolution for this case than clearing the basis after a certain number of ``gmres`` iterations.
+
+* The ``relative residual`` is set to 1e-4. This is small enough, so we don't under-resolve our matrix and do extra non-linear steps because of it, and at the same, it doesn't require too many ``gmres`` iterations.
```

Comment:
```suggestion
* The ``relative residual`` is set to 1e-4. This is small enough, so we do not under-resolve our matrix and do extra non-linear steps because of it, and at the same, it doesn't require too many ``gmres`` iterations.
```